package.json

```diff
@@ -5,11 +5,11 @@
   "scripts": {
     "start": "NODE_ENV=development node devServer.js",
     "start:production": "NODE_ENV=production node devServer.js",
-    "build:development": "POET_ENV=development webpack",
-    "build:testing": "POET_ENV=testing webpack",
+    "build:development": "POET_ENV=development webpack --mode=production",
```

Comment:
uhmm I'm not sure, because this command generates a new `build`.

Reply:
what do you say? what benefits does it bring in the build if I change to production?

Reply:
I think it makes development harder/slower because it runs a bunch of plugins:
https://webpack.js.org/concepts/mode/

I think setting it to "development" makes sense for the 'build:development' script.